docs/benchmarks/driving_smarts_2023_3.rst

```diff
@@ -0,0 +1,254 @@
+      + :attr:`~smarts.core.agent_interface.AgentInterface.waypoint_paths`. 
+
+4. inference/MANIFEST.in 
+    + Contains any file paths to be included in the package.
+
+5. inference/setup.cfg
+    + Must contain the following template code. 
+    + The template code helps build the user policy into a Python package.
+    
+      .. code-block:: cfg
+
+        [metadata]
+        name = <...>
+        version = 0.1.0
+        url = https://github.com/huawei-noah/SMARTS
+        description = SMARTS zoo agent.
+        long_description = <...>. See [SMARTS](https://github.com/huawei-noah/SMARTS).
+        long_description_content_type=text/markdown
+        classifiers=
+            Programming Language :: Python
+            Programming Language :: Python :: 3 :: Only
+            Programming Language :: Python :: 3.8
+
+        [options]
+        packages = find:
+        include_package_data = True
+        zip_safe = True
+        python_requires = == 3.8.*
+        install_requires = 
+            <...>==<...>
+            <...>==<...>
+
+    + User may fill in the ``<...>`` spaces in the template.
+    + User should provide a name for their policy in the ``long_description`` section.
+
+6. inference/setup.py
+    + Keep this file and its default contents unchanged.
+    + Its default contents are shown below.
+
+      .. code-block:: python
+    
+        from setuptools import setup
+
+        if __name__ == "__main__":
+            setup()
+ 
+Example
+-------
+
+An example training and inference code is provided for this benchmark. 
+See the :examples:`rl/platoon` example. The example uses PPO algorithm from 
+`Stable Baselines3 <https://github.com/DLR-RM/stable-baselines3>`_ reinforcement learning library. 
+Instructions for training and evaluating the example is as follows.
+
+Train
+^^^^^
++ Setup
+
+  .. code-block:: bash
+
+    # In terminal-A
+    $ cd <path>/SMARTS/examples/rl/platoon
+    $ python3.8 -m venv ./.venv
+    $ source ./.venv/bin/activate
+    $ pip install --upgrade pip wheel
+    $ pip install -e ./../../../.[camera_obs,argoverse]
+    $ pip install -e ./inference/
+
++ Train locally without visualization
+
+  .. code-block:: bash
+
+    # In terminal-A
+    $ python3.8 train/run.py
+
++ Train locally with visualization
+
+  .. code-block:: bash
+
+    # In terminal-A
+    $ python3.8 train/run.py --head
+
+  .. code-block:: bash
+
+    # In a different terminal-B
+    $ scl envision start
+    # Open http://localhost:8081/
+
++ Trained models are saved by default inside the ``<path>/SMARTS/examples/rl/platoon/train/logs/`` folder.
+
+Docker
+^^^^^^
++ Train inside docker
+
+  .. code-block:: bash
+
+    $ cd <path>/SMARTS
+    $ docker build --file=./examples/rl/platoon/train/Dockerfile --network=host --tag=platoon .
+    $ docker run --rm -it --network=host --gpus=all platoon
+    (container) $ cd /SMARTS/examples/rl/platoon
```

Comment:
Currently the Dockerfile sets `WORKDIR /SMARTS` , which I try to make consistent across various dockerfiles. This helps when we have examples in different paths, as the dockerfile can be built similarly and we only need to alter the command issued.

I simply reiterated the full path here `$ cd /SMARTS/examples/rl/platoon` to avoid any confusion for the users.